### ISSUE TITLE
Unexpected SystemSettings interaction causing failure to run via wsgi.py

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#11660]
@ekansa reported on the [forum](https://community.archesproject.org/t/arches-via-docker-installation-issues/2603/9?u=jacobtylerwalls) a failure to run an arches project via `wsgi.py`, i.e. not run with `runserver`. The TL;DR is that we have the default value of `APP_NAME` being stored in a system settings tile and overriding the project settings file, only under wsgi.py.

Traced it to this unexpectedly evaluating True:
https://github.com/archesproject/arches/blob/f3390eff2ff5fe8eca899d2b3f199868fcea1575/arches/urls.py#L789

... because wsgi.py loads system settings from the database:
https://github.com/archesproject/arches/blob/f3390eff2ff5fe8eca899d2b3f199868fcea1575/arches/wsgi.py#L39

... and we save the value "Arches" for `APP_NAME` in the database during `setup_db`, which is going to be incorrect for all cases (only applies to running "naked" core arches like a project):
https://github.com/archesproject/arches/blob/f3390eff2ff5fe8eca899d2b3f199868fcea1575/arches/db/system_settings/Arches_System_Settings.json#L153-L156
https://github.com/archesproject/arches/blob/f3390eff2ff5fe8eca899d2b3f199868fcea1575/arches/management/commands/setup_db.py#L234-L236

***
@chrabyrd and I didn't notice this when developing f569e097f02c1973a9b0c216bf9902a6f19d6e8c and related features because we were running the devserver, but arguably this is an orthogonal bug.

We have other cases where we need the value of APP_NAME to be correct:

https://github.com/archesproject/arches/blob/f3390eff2ff5fe8eca899d2b3f199868fcea1575/arches/settings_utils.py#L161-L162

***
I reproduced on 7.6.3 with Eric's instructions:

- clone opencontext/arches-via-docker
- checkout the 'local' branch
- cp edit_dot_env .env
- make DEBUG=False (in .env) and also change the SHOW_LANGUAGE_SWITCH to be True
- commented out `DEPLOY_HOST` in settings_local since there wasn't a fallback
- prepend `npm i typescript@5.6.3` to the build development command to work around https://github.com/vuejs/language-tools/issues/5018. (This will hopefully disappear in a few days.)
- visit the server at 0.0.0.0:8004

#### Next steps

A workaround is to comment out the last line of wsgi.py that calls `update_from_db()`, or updating the invalid tile value in pg directly. But regardless of the wisdom of that, there a few things to look into, some of which shouldn't be done in a patch?
- fix the values in these tiles (migration? in v8?)
- update the python SystemSettings to ignore this node (can do in a patch)
- stop creating values in these tiles (in v8?)

***
cc/ @aarongundel

**Comment by ekansa** [comment]
@jacobtylerwalls thanks for the deep investigation and diagnosis!

**Comment by jacobtylerwalls** [comment]
Throwing out ideas for better heuristics for _"is the currently running Django project core arches itself?"_:

this should only evaluate True for running core arches, but it's a little fiddly:
```py
>>> from pathlib import Path
>>> from django.conf import settings
>>> Path(settings.APP_ROOT).parent == Path(settings.ROOT_DIR)
True
```

**Comment by jacobtylerwalls** [comment]
Also worth looking into whether core urls.py can just do the same check that's working well in projects?
```py
if settings.ROOT_URLCONF == __name__:
```

**Comment by jacobtylerwalls** [comment]
I'll open follow-ups for anything not addressed in 7.6.4.
